daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/preprocessing/ValueTranslator.scala

```diff
@@ -6,14 +6,14 @@ package engine
 package preprocessing
 
 import com.daml.lf.data._
+import com.daml.lf.data.Ref._
```

Comment:
please do use wildcard only if you need to import 6 or more entities or implicits. 


(use the following suggestion as you wish)
Also I personally like using qualified (or partially qualify) entities, in your example, I will import 
````
import com.daml.lf.data.Ref
```
and use `Ref.Name` in the body. Like that you make it clear which name you are speaking about.

Reply:
Fair point, will fix.